dotnet/samples/Demos/TelemetryWithAppInsights/Program.cs

```diff
@@ -294,8 +294,14 @@ private sealed class AIServiceSelector : IAIServiceSelector
                             Temperature = 0,
                             ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions
                         },
-                        GoogleAIGeminiServiceKey => new GeminiPromptExecutionSettings(),
-                        HuggingFaceServiceKey => new HuggingFacePromptExecutionSettings(),
+                        GoogleAIGeminiServiceKey => new GeminiPromptExecutionSettings()
+                        {
+                            Temperature = 0,
+                        },
```

Comment:
What's the reason for not including tool calling here?

Reply:
I found an issue with Gemini when invoking a Kernel function with auto invoke enabled. I will file an issue in a moment.

Reply:
Ok. I suggest we leave the line in but commented out and with the issue link next to it... it can then be uncommented when the issue is fixed (hopefully soon :smile:)